src/screens/Exchange/Exchange.js

```diff
@@ -383,7 +384,7 @@ function SelectorInputTemplate(locals) {
   );
 }
 
-function getCardTopButtonData(topButtonProps) {
+function getCardAdditionalButtonData(additionalData) {
```

Comment:
nit: this file is way too long already, and most of these helper functions could be moved to utils. Also it wouldn't hurt to check type here. All this should be done as a separate task though I guess. Any thoughts on this?

Reply:
If the function returns data that's presentation (like titles, press handlers, etc), it shouldn't be on utls, IMHO. I'd try to separate the code into components. Not entirely sure if this specific case could become its own component, but if not, there are parts that can.

Reply:
Yeah, you are both right :)
File is in deed long, I'd also be up for leaving helper functions next to the rest of related code when it is not globally used and I was thinking about dividing the file into two parts - one for the form and selectors and one for the offers list.

Dmitry has asked me to fix scrolling layout here, so I could divide this file when doing so :)